### DISCUSSION TITLE
where to get IfcOpenShell-python ubuntu version 0.6.0b0

### DISCUSSION THREAD

**Discussion by lwang64** [IfcOpenShell/IfcOpenShell#3814] [Q&A]
Hi, does anyone know where I can get the IfcOpenShell-python ubuntu version 0.6.0b0. I can only access 0.7 now on the website.

Thanks very much.

**Comment by aothms** [comment]
https://github.com/IfcOpenBot/IfcOpenShell/commit/721fe4729aa5302efe1602971aae2558934ad098#comments is the latest and most stable 0.6 build. I don't remember if it is -b0 precisely.

**Reply by Moult** [reply]
@aothms what do you think about me adding this to the documentation under an "older versions" section? is it worth it? what about <=0.5 ?

**Reply by aothms** [reply]
I wouldn't recommend using them tbh... When we get to general availability of 0.8 things will likely be unstable for a while.
